src/libs/elektra/keyset.c

```diff
@@ -380,7 +384,15 @@ int ksDel (KeySet * ks)
 int ksClear (KeySet * ks)
 {
 	ksClose (ks);
-	// ks->array empty now
+// ks->array empty now
+
+#ifdef ELEKTRA_ENABLE_OPTIMIZATIONS
+	ks->opmphm = opmphmNew ();
+	if (!ks->opmphm)
+	{
+		return -1;
```

Comment:
Currently I defined that the `opmphm` is empty when all `size[]` are zero, I see no reason to use `calloc()`, since the `opmphmNew()` sets the sizes to zero. All other fields of the Opmphm get initialized during build.